src/main/java/com/team766/orin/GetOrinRawValue.java

```diff
@@ -0,0 +1,131 @@
+package com.team766.orin;
+
+import edu.wpi.first.networktables.DoubleArrayEntry;
+import edu.wpi.first.networktables.DoubleArrayTopic;
+import edu.wpi.first.networktables.IntegerArrayEntry;
+import edu.wpi.first.networktables.IntegerArrayTopic;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+/*
+ * This class is used to get raw values from the NetworkTable. We can send values on the Orin to NetworkTables and get them easily here.
+ * One important caveat of the NetworkTables code is that it doesn't throw exceptions when a value is not found.
+ * Instead, it asks for a default value to return. This is a problem because we can't know if the value is not found or if the value is the default value.
+ * Thus, numbers were chosen as default values that aren't really plausible to be sent by the Orin.
+ * For example, it is not likley that the Orin will send Integer.MIN_VALUE as a value. In this case, an exception is thrown.
+ *
+ * @author Max Spier, 9/28/2024
+ */
+public class GetOrinRawValue {
+
+    private static NetworkTableInstance inst = NetworkTableInstance.getDefault();
```

Comment:
Why would we want to have an instance of this class? The design of this class is to be a utility class for the first layer of getting data from NetworkTables to the robot. IMO I don't think it would be good to make it non-static if we are going to want different parts of code using it at the same time.

Reply:
Ah! I'll take a look tonight at the meeting! Thanks!